commands.js

```diff
@@ -2183,6 +2183,18 @@ var commands = exports.commands = {
 		user.team = target;
 	},
 
+	vtm: function (target, room, user, connection) {
+		var format = Tools.getFormat(target);
+		if (!format.name) format = Tools.getFormat('Anything Goes');
```

Comment:
Also another existence check should be done afterwards, for servers could remove the Anything Goes format.